static/docs/tutorial/index.md

```diff
@@ -25,7 +25,7 @@ and this approach will not require storing binary files in your Git repository.
 
 ## DVC Workflow
 
-The diagram below describes all the DVC commands and relationships between local
-cache and remote storage.
+The diagram below describes all the DVC commands and relationships between a
+local cache and remote storage.
```

Comment:
local - forgot to remove?

Reply:
No, this one I left on purpose because the image below reads "Local Cache":

![image](https://user-images.githubusercontent.com/1477535/64368681-6b388180-cfe0-11e9-8ab3-01c6a6c7096e.png)

Reply:
👍